package.json

```diff
@@ -44,14 +44,15 @@
     "underscore": "^1.9.2"
   },
   "devDependencies": {
+    "@mixmaxhq/jest-coverage-config": "^2.0.1",
+    "@mixmaxhq/jest-junit-config": "^1.0.0",
```

Comment:
These are currently private, and I imagine we don't want to block external folks from `npm i`/`npm ci`.